packages/ember-data/lib/system/store.js

```diff
@@ -1073,6 +1073,7 @@ Store = Service.extend({
     }
 
     typeMap.findAllCache = null;
+    this.typeMapFor(type).metadata = {};
```

Comment:
Changed.  Are there docs somewhere on Ember.create ? It's not in the website api docs. I'm assuming this is the underlying instantiation method (similar to Ember.set or Ember.get) that get's added to Ember.Object?